test/node.js

```diff
@@ -0,0 +1,126 @@
+import { expect } from "chai";
+import fs from "fs";
+import os from "os";
+import path from "path";
+import process from "process";
+import { promisify } from "util";
+
+describe("Errors", function () {
+  it("Should return error", function () {
+    function throwError() {
+      throw new Error("oh no!");
+    }
+    expect(() => throwError()).to.throw("oh no!");
+  });
+});
+
+describe("OS", function () {
+  it("Should use os.arch()", function () {
+    expect(os.arch()).to.equal("x64");
+  });
+  it("Should use os.homedir()", function () {
+    console.log(os.homedir());
+  });
+});
+
+describe("Path", function () {
+  it("Should use path.basename()", function () {
+    expect(path.basename("/foo/bar/baz/asdf/quux.html")).to.equal("quux.html");
+  });
+  it("Should use path.dirname()", function () {
+    expect(path.dirname("/foo/bar/baz/asdf/quux.html")).to.equal(
+      "/foo/bar/baz/asdf"
+    );
+  });
+  it("Should use path.format()", function () {
+    const pathObj = {
+      root: "/ignored",
+      dir: "/home/user/dir",
+      base: "file.txt",
+    };
+    expect(path.format(pathObj)).to.equal("/home/user/dir/file.txt");
+  });
+  it("Should use path.isAbsolute()", function () {
+    expect(path.isAbsolute("/foo/bar")).to.equal(true);
+    expect(path.isAbsolute("/baz/..")).to.equal(true);
+    expect(path.isAbsolute("qux/")).to.equal(false);
+    expect(path.isAbsolute(".")).to.equal(false);
+  });
+});
+
+describe("File System", function () {
+  const unlinkAsync = promisify(fs.unlink);
+  const renameAsync = promisify(fs.rename);
+
+  const fileContent = "Hello content!";
+  const currentFolder = process.cwd();
+  const file1 = currentFolder + "/test-folder/fs_test_file.txt";
+  const file2 = currentFolder + "/test-folder/fs_test_file1.txt";
```

Comment:
Try using `path.join` here instead of concatenating strings.

To learn multiple approaches, try using `__dirname` as the root of the path for file2.

Reply:
Ya, that's expected. :+1: